.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: true,
+    trailingComma: "all",
+    singleQuote: true,
+    printWidth: 120,
+    tabWidth: 4
```

Comment:
Could we switch `tabWidth` to `2`? It's extremely weird to use 4, as the industry is set on `2`.

Reply:
Fixed in 719acd3bc1482baa9ce7961be63e21c6b04590a4